plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1278,25 +1267,45 @@ export default class ChatLivePane extends Component {
 
   @action
   computeDatesSeparators() {
+    throttle(this, this._computeDatesSeparators, 50, false);
+  }
+
+  _computeDatesSeparators() {
     schedule("afterRender", () => {
       const dates = [
         ...this._scrollerEl.querySelectorAll(".chat-message-separator-date"),
       ].reverse();
-      const scrollHeight = this._scrollerEl.scrollHeight;
+      const height = this._scrollerEl.querySelector(
+        ".chat-messages-container"
+      ).clientHeight;
 
       dates
         .map((date, index) => {
-          const item = { bottom: "0px", date };
+          const item = { bottom: 0, date };
```

Comment:
I switched to height and position from bottom to avoid a bug where it would for a scroll to top